installer.c

```diff
@@ -1086,6 +1090,24 @@ EFI_STATUS efi_main(EFI_HANDLE image, EFI_SYSTEM_TABLE *_table)
 		}
 	}
 
+	/* Output log to the installer.log if run the command */
+	if (options) {
+		if (!strncmp(options, "-l", 2)) {
+			KERNELFLINGER_LOG_OUTPUT = 1;
+			options += 2;
+			skip_whitespace((char **)&options);
+		}
+		if (flag_installer_log_file == FALSE) {
```

Comment:
**Coding style violation**:
It should be 
`if (!flag_installer_log_file && KERNELFLINGER_LOG_OUTPUT) {`

Reply:
Already modified.